transforms/sort_test.go

```diff
@@ -61,56 +93,59 @@ var sortTests = []struct {
 	},
 }
 
-func TestSort(t *testing.T) {
-	for _, sortTest := range sortTests {
-		for _, sort := range []optimus.TransformFunc{Sort(sortTest.less), StableSort(sortTest.less)} {
+func TestSortError(t *testing.T) {
```

Comment:
Can we add an error test for your new sort method?

Reply:
No - https://github.com/Clever/optimus/pull/60/files#diff-a513864350facb09df05bd63daec0b00R71